client/src/utils/prettifyStatus.js

```diff
@@ -5,12 +5,20 @@ import { Button } from '../components/generic';
 
 export const prettifyStatus = (status, id, tabValue, handleEngage) => {
   let firstStatus = status[0];
+  let isWithdrawn = false;
   if (status[0] === 'offer_made') firstStatus = 'Offer Made';
   if (status[0] === 'open') firstStatus = 'Open';
   if (status[0] === 'prospecting') firstStatus = 'Prospecting';
   if (status[0] === 'interviewing') firstStatus = 'Interviewing';
   if (status[0] === 'rejected') firstStatus = 'Archived';
   if (status[0] === 'hired') firstStatus = 'Hired';
+  if (status.includes('withdrawn')) {
```

Comment:
```suggestion
  if (status[1] === 'withdrawn') {
```

Reply:
https://github.com/bcgov/hcap/blob/master/server/services/participants.js#L185

Reply:
How can we be so confident in the indexes? Why are the others 0 and this one 1?